rpc/proto/mayastor.proto

```diff
@@ -52,6 +52,7 @@ enum ShareProtocol {
   NONE = 0;   // not exposed
   NVMF = 1;   // NVMe over Fabrics (TCP)
   ISCSI = 2;  // iSCSI
+  NBD = 3;    // NBD
```

Comment:
If we are using the ShareProtocol for the front-end as well as the back-end, to comment above should be expanded? It is not just exposing a replica.

Reply:
Fixed by https://github.com/tjoshum/Mayastor/pull/1/commits/b13cb10b7a576afc8d1bbe8822a45eaf5368e0a4